frontend/public/models/index.ts

```diff
@@ -814,6 +814,45 @@ export const VirtualMachineInstancePresetModel: K8sKind = {
   id: 'virtualmachineinstancepreset',
 };
 
+export const DataVolume: K8sKind = {
```

Comment:
This should go to kubevirt models `frontend/public/kubevirt/models/vm.ts`.
Once moved, we should merge this after https://github.com/kubevirt/web-ui/pull/120

Reply:
done